ts/private/ts_project_worker.js

```diff
@@ -3,8 +3,14 @@ const ts = require('typescript');
 const path = require('path');
 const worker = require('@bazel/worker');
 const v8 = require('v8');
+
 const MNEMONIC = 'TsProject';
 
+// Workaround for https://github.com/microsoft/TypeScript/pull/42095
```

Comment:
why do we need a workaround if that PR was merged almost 2 years ago?

Reply:
these changes are the cause of the bug, not the solution.

Reply:
ah, then how about "workaround the issue introduced in ..." and ideally also link to (or create) an issue describing what they broke

Reply:
that's better. I'll file the issue with no-hope-to-see-it-fixed.